File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L4/06/fungsi.kt
package IR

import java.util.Scanner

fun main() {
    println("Enter an integer: ")
    val sc = Scanner(System.`in`)
    var input = sc.nextInt()
    while (input != 0) {
        val sisa = input % 10
        print(sisa)
        input /= 10
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L6/06/fungsi.ts
import { ArrayList, List } from 'typescript-collections';

function terbalik(angka: number): void {
  const arr: List<number> = new ArrayList<number>();
  while (angka > 0) {
    const sisa: number = angka % 10;
    arr.add(sisa);
    angka = Math.floor(angka / 10);
  }
  for (let i = 0; i < arr.size(); i++) {
    console.log(arr.get(i));
  }
  console.log();
}

function main(): void {
  console.log('Enter an integer: ');
  const sc: Scanner = new Scanner(System.in);
  const input: number = sc.nextInt();
  terbalik(input);
}

main();

